queries.sql

```diff
@@ -1,3 +1,31 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals
+WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+
+SELECT name FROM animals
+WHERE date_of_birth
+BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Querying all animals born between 2016 and 2019, should be similar to:
```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-31-12';
```
Let's improve upon our query to match the above.